chain/westend/defaults.go

```diff
@@ -29,6 +29,8 @@ func DefaultConfig() *cfg.Config {
 	config.Core.GrandpaAuthority = false
 	config.Core.Role = 1
 	config.Network.NoMDNS = false
+	config.Log.Digest = "trace"
```

Comment:
was this used for testing or do we really need the `trace` log level?

Reply:
I had missed out on the module log-level parsing. I've added it here #3285

Reply:
Yeah, I forget to remove this delta before committing, btw thanks for the new flags!